packages/expo-in-app-purchases/src/InAppPurchases.ts

```diff
@@ -234,6 +236,7 @@ export function setPurchaseListener(
       callback(result);
     }
   );
+  return () => purchaseUpdatedSubscription.remove();
```

Comment:
I think it's more common to return the entire subscriber, e.g. `expo-linking`

Reply:
Ah, yes, good point.

Reply:
Please make sure to test your change too and include a test plan